app/controllers/fhir/r4/api_controller.rb

```diff
@@ -27,6 +27,7 @@ class Fhir::R4::ApiController < ActionController::API
     )
   end
   before_action :check_client_type
+  rescue_from StandardError, with: :handle_server_error
```

Comment:
I like this idea a lot 👍

Reply:
I didn't know about this callback, this is nice